apps/omg/lib/omg/state/measurement_calculation.ex

```diff
@@ -35,13 +35,19 @@ defmodule OMG.State.MeasurementCalculation do
 
   defp unique_users(utxos) do
     utxos
-    |> Enum.map(fn {_, %OMG.Utxo{owner: owner}} -> owner end)
+    |> Enum.map(fn {_, %OMG.Utxo{output: output}} -> output end)
+    |> Enum.map(&Map.get(&1, :owner))
```

Comment:
`:owner` is not a part of Output protocol. As of now that's not a problem as we have only `FungibleMoreVPToken` struct. Just saying.